Bottom: 91fc28614e8a39786bdcb28210e3cc976da7e333
Top:    752a10e5d3ea2344a8257cb1fd5f6f984c252759
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-18 20:37:43 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
